global/matlab/icaDim/icaDim.m

```diff
@@ -1,9 +1,19 @@
-function [Out] = icaDim(Origdata,DEMDT,VN,Iterate,NDist)
+function [Out] = icaDim(Origdata, DEMDT, VN, Iterate, NDist)
+%
+% ICADIM(Origdata, DEMDT, VN, Iterate, NDist)
+%
+% Estimates the number of ica components that are not due to random noise,
+%   and makes a standard deviation map of the random noise estimate
+%
+% DEMDT: 1 to detrend, 0 to only demean, -1 to do nothing
+% VN: initial variance normalization dimensionality
+% Iterate: mode/count for iterating the estimation
```

Comment:
These make sense, except `Iterate` remains ambiguous. e.g., what are the different modes?  What does -1 represent?

Reply:
It is complicated, ad-hoc, and shouldn't really be combined into a single input argument, so it may not deserve description.

Reply:
Even if ad-hoc, it would be helpful to have _something_, short of having to decipher the code itself.

Reply:
Looks like 0 will crash, anything negative triggers one block of code that messes with the loop variables to cause an exit based on averaging all the dimensionality estimates after iteration 3 (also unfortunately changing what the value of stabThresh means), 1-3 do the loop as written (looks like number of agreements on dimensionality, without needing to be consecutive), and > 3 messes with the loop variables to cause it to iterate exactly that many times.  This was already summarized in a comment about 5 lines below the comments I added.  Some of these modes may not get used anyway...

Reply:
So, that comment ~5 lines later in the code, is almost sufficient for a high-level overview of what is going on, but needs to include:
1. A brief explanation of the difference between "legacy" vs "new" convergence mode.
2. Explanation of what "1 for VNDIM=1 results" means.  (Does it just mean 1 iteration and then done?)

The inclusion of that extra bit of info, and then moving the whole explanation up to the top, would make me happy :)